README.adoc

```diff
@@ -2,188 +2,184 @@
+. Click *Dependencies* and select *Eureka Server* for the server application. Select *Eureka Discovery Client* and *Spring Web* for the client applications.
 . Click *Generate*.
 . Download the resulting ZIP file, which is an archive of a web application that is configured with your choices.
 
 NOTE: If your IDE has the Spring Initializr integration, you can complete this process from your IDE.
 
-NOTE: You can also fork the project from Github and open it in your IDE or other editor.
-
 [[initial]]
 == Start a Eureka Service Registry
 
-You first need a Eureka Service registry. You can use Spring Cloud's `@EnableEurekaServer`
+You first need a Eureka Server. You can use Spring Cloud's `@EnableEurekaServer`
 to stand up a registry with which other applications can communicate. This is a regular
 Spring Boot application with one annotation (`@EnableEurekaServer`) added to enable the
 service registry. The following listing (from
-`eureka-service/src/main/java/com.example.serviceregistrationanddiscoveryservice/ServiceRegistrationAndDiscoveryServiceApplication.java`)
-shows the service application:
+`eureka-server/src/main/java/com/example/eurekaserver/EurekaServerApplication.java`)
+shows the server application:
 
 ====
 [source,java]
 ----
-include::complete/eureka-service/src/main/java/com/example/serviceregistrationanddiscoveryservice/ServiceRegistrationAndDiscoveryServiceApplication.java[]
+include::eureka-server/src/main/java/com/example/eurekaserver/EurekaServerApplication.java[]
 ----
 ====
 
-When the registry starts, it will complain (with a stacktrace) that there are no replica
-nodes to which the registry can connect. In a production environment, you will want more
-than one instance of the registry. For our simple purposes, however, it suffices to
-disable the relevant logging.
+In a production environment, you will likely want more than one instance of the registry.
+You can find additional information about configuring the Eureka Server https://docs.spring.io/spring-cloud-netflix/reference/spring-cloud-netflix.html#spring-cloud-eureka-server[here^].
 
 By default, the registry also tries to register itself, so you need to disable that
-behavior as well.
-
-It is a good convention to put this registry on a separate port when using it locally.
+behavior. Additionally, it is a good convention to put this registry on a separate port when using it locally.
 
-Add some properties to `eureka-service/src/main/resources/application.properties` to
-handle all of these requirements, as the following listing shows:
+Add some properties to `eureka-server/src/main/resources/application.properties` to
+handle these requirements, as the following listing shows:
 
 ====
 [source,properties]
 ----
-include::complete/eureka-service/src/main/resources/application.properties[]
+include::eureka-server/src/main/resources/application.properties[]
 ----
 ====
 
+You can now run the Eureka Server with the command `./gradlew bootRun`.
```

Comment:
Change to:

You can now start the Eureka server by running `./gradlew bootRun`.